{"uid":"fe7cf1a26ac328c7","name":"test_recipe_create['Сало в соевом соусе вареное'-'60'-'Потрясающе ароматное и мягкое,                                 никто не сможет устоять!                                 Сало в соевом соусе вареное делается очень легко и просто!                                 Его можно предложить гостям на праздничный стол                                 в виде закуски или подать на семейный ужин.                                 Оно прекрасно сочетается со свежими овощами                                 или салатами. Обязательно пробуем!'-'Ужин'-{'сало': '250', 'вода': ' 200', 'соевый': '250', 'чеснок': '15'}-'salo-v-soevom-souse-varenoe.jpg']","fullName":"tests.test_recipe_create_page.TestRecipeCreate#test_recipe_create","historyId":"3f1a1b7b3cc23e3abbdb4f7b6db452ad","time":{"start":1689186458363,"stop":1689186458363,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55d7f03754e3 <unknown>\n#1 0x55d7f00a4c76 <unknown>\n#2 0x55d7f00cdd78 <unknown>\n#3 0x55d7f00ca029 <unknown>\n#4 0x55d7f0108ccc <unknown>\n#5 0x55d7f010847f <unknown>\n#6 0x55d7f00ffde3 <unknown>\n#7 0x55d7f00d52dd <unknown>\n#8 0x55d7f00d634e <unknown>\n#9 0x55d7f03353e4 <unknown>\n#10 0x55d7f03393d7 <unknown>\n#11 0x55d7f0343b20 <unknown>\n#12 0x55d7f033a023 <unknown>\n#13 0x55d7f03081aa <unknown>\n#14 0x55d7f035e6b8 <unknown>\n#15 0x55d7f035e847 <unknown>\n#16 0x55d7f036e243 <unknown>\n#17 0x7f38cdda8b43 <unknown>","statusTrace":"E   selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n      (unknown error: DevToolsActivePort file doesn't exist)\n      (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n    Stacktrace:\n    #0 0x55d7f03754e3 <unknown>\n    #1 0x55d7f00a4c76 <unknown>\n    #2 0x55d7f00cdd78 <unknown>\n    #3 0x55d7f00ca029 <unknown>\n    #4 0x55d7f0108ccc <unknown>\n    #5 0x55d7f010847f <unknown>\n    #6 0x55d7f00ffde3 <unknown>\n    #7 0x55d7f00d52dd <unknown>\n    #8 0x55d7f00d634e <unknown>\n    #9 0x55d7f03353e4 <unknown>\n    #10 0x55d7f03393d7 <unknown>\n    #11 0x55d7f0343b20 <unknown>\n    #12 0x55d7f033a023 <unknown>\n    #13 0x55d7f03081aa <unknown>\n    #14 0x55d7f035e6b8 <unknown>\n    #15 0x55d7f035e847 <unknown>\n    #16 0x55d7f036e243 <unknown>\n    #17 0x7f38cdda8b43 <unknown>","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"browser","time":{"start":1689186458364,"stop":1689186459017,"duration":653},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55d7f03754e3 <unknown>\n#1 0x55d7f00a4c76 <unknown>\n#2 0x55d7f00cdd78 <unknown>\n#3 0x55d7f00ca029 <unknown>\n#4 0x55d7f0108ccc <unknown>\n#5 0x55d7f010847f <unknown>\n#6 0x55d7f00ffde3 <unknown>\n#7 0x55d7f00d52dd <unknown>\n#8 0x55d7f00d634e <unknown>\n#9 0x55d7f03353e4 <unknown>\n#10 0x55d7f03393d7 <unknown>\n#11 0x55d7f0343b20 <unknown>\n#12 0x55d7f033a023 <unknown>\n#13 0x55d7f03081aa <unknown>\n#14 0x55d7f035e6b8 <unknown>\n#15 0x55d7f035e847 <unknown>\n#16 0x55d7f036e243 <unknown>\n#17 0x7f38cdda8b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/testing_foodgram_project_react/testing_foodgram_project_react/conftest.py\", line 26, in browser\n    browser = webdriver.Chrome(\n              ^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 84, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 104, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"feature","value":"Тесты создания рецепта."},{"name":"story","value":"Тест создания рецепта."},{"name":"tag","value":"recipe_create"},{"name":"tag","value":"recipe_create_test(scope='function')"},{"name":"tag","value":"run(order=3)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_recipe_create_page"},{"name":"subSuite","value":"TestRecipeCreate"},{"name":"host","value":"fv-az256-210"},{"name":"thread","value":"2518-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_recipe_create_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"cooking_time_data","value":"'60'"},{"name":"image_name_data","value":"'salo-v-soevom-souse-varenoe.jpg'"},{"name":"ingredients_data","value":"{'сало': '250', 'вода': ' 200', 'соевый': '250', 'чеснок': '15'}"},{"name":"meal_data","value":"'Ужин'"},{"name":"recipe_description_data","value":"'Потрясающе ароматное и мягкое,                                 никто не сможет устоять!                                 Сало в соевом соусе вареное делается очень легко и просто!                                 Его можно предложить гостям на праздничный стол                                 в виде закуски или подать на семейный ужин.                                 Оно прекрасно сочетается со свежими овощами                                 или салатами. Обязательно пробуем!'"},{"name":"recipe_name_data","value":"'Сало в соевом соусе вареное'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=3)","recipe_create","recipe_create_test(scope='function')"]},"source":"fe7cf1a26ac328c7.json","parameterValues":["'60'","'salo-v-soevom-souse-varenoe.jpg'","{'сало': '250', 'вода': ' 200', 'соевый': '250', 'чеснок': '15'}","'Ужин'","'Потрясающе ароматное и мягкое,                                 никто не сможет устоять!                                 Сало в соевом соусе вареное делается очень легко и просто!                                 Его можно предложить гостям на праздничный стол                                 в виде закуски или подать на семейный ужин.                                 Оно прекрасно сочетается со свежими овощами                                 или салатами. Обязательно пробуем!'","'Сало в соевом соусе вареное'"]}